src/components/text-editor/text-editor.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+    Component,
+    Element,
+    Event,
+    EventEmitter,
+    State,
+    h,
+} from '@stencil/core';
+import { EditorState } from 'prosemirror-state';
+import { EditorView } from 'prosemirror-view';
+import { Schema, DOMParser } from 'prosemirror-model';
+import { schema } from 'prosemirror-schema-basic';
+import { exampleSetup } from 'prosemirror-example-setup';
+
+/**
+ * The `limel-text-editor` component wraps the toast-ui react editor. This editor
+ * allows a rich text editing experience with markdown support.
+ *
+ * @exampleComponent limel-example-text-editor
+ * @exampleComponent limel-example-text-editor-prefilled-value
+ * @beta
+ */
+@Component({
+    tag: 'limel-text-editor',
+    shadow: true,
+    styleUrl: 'text-editor.scss',
+})
+export class TextEditor {
+    @Element()
+    private host: HTMLLimelTextEditorElement;
+
+    @State()
+    private view: EditorView;
+
+    @Event()
+    private change: EventEmitter<{ html: string }>;
+
+    public componentWillLoad() {}
+
+    public render() {
+        return [<div id="editor" />, <div id="content" />];
```

Comment:
the generated HTML looks like this:
<img width="319" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/46ef3432-279a-40b9-b292-3fbc011129c9">

1. The `<div class="ProseMirror-menubar-wrapper">` contains the content too, and the `<div id="content"></div>` is rendered empty. So something is wrong here. 

When I check [their example](https://prosemirror.net/) they don't seem to have this `<div id="content">`. Why do WE have it?

Reply:
Yeah it seems okay to not have it. The tutorial does it differently but everything continues working (and I think the button clicking problem you mentioned below even goes away?)

Reply:
yes, the button-clicking problem no longer exists!